cypress/integration/use-cases/migration.spec.js

```diff
@@ -34,29 +34,22 @@ const complaint = {
 
 describe('Saved requests in the legacy database should be correctly migrated', () => {
     beforeEach(() => {
-        cy.visit('/');
-        // eslint-disable-next-line cypress/no-unnecessary-waiting
-        cy.wait(500);
-
-        // Clear the persisted data so that migration will start again
-        cy.window().then((win) => win.accessPrivacyAsyncStorage('proceedings').clear());
+        // Set IndexedDB to the state localforage creates
+        cy.clearIndexedDb('Datenanfragen.de');
+        cy.openIndexedDb('Datenanfragen.de').as('dadeDB').createObjectStore('id-data');
+        cy.getIndexedDb('@dadeDB').createObjectStore('local-forage-detect-blob-support');
+        cy.getIndexedDb('@dadeDB').createObjectStore('wizard-companies');
+        cy.getIndexedDb('@dadeDB').createObjectStore('my-requests').as('myRequestsStore');
     });
 
-    it('Migrate a normal proceeding', () => {
+    it('Migrates a normal proceeding', () => {
         // Add test data
-        cy.window().then((win) =>
-            win
-                .accessLocalForageStore('my-requests')
-                .setItem(`${reference}-access`, accessRequest)
-                .then(() =>
-                    win.accessLocalForageStore('my-requests').setItem(`${reference}-custom-admonition`, admonition)
-                )
-                .then(() =>
-                    win.accessLocalForageStore('my-requests').setItem(`${reference}-custom-complaint`, complaint)
-                )
-        );
+        cy.getStore('@myRequestsStore')
+            .createItem(`${reference}-custom-admonition`, admonition)
+            .createItem(`${reference}-access`, accessRequest)
```

Comment:
Is the order here deliberate?

Reply:
Yep, because the code should work even though the OR comes after the other messages in the database.